lightbluetent/home.py

```diff
@@ -145,9 +164,9 @@ def register():
     else:
         # defaults
         values = {
-            "first_name": "First name from lookup",
-            "surname": "Surname from lookup",
-            "email_address": "Email address from lookup",
+            "first_name": "",
+            "surname": "",
+            "email_address": "",
```

Comment:
Right call to remove these defaults: you could replace with the `placeholder` attribute in the HTML input tags, e.g. `<input placeholder="First name">`

Reply:
Resolved by 79700e3.